integrated_channels/moodle/transmitters/content_metadata.py

```diff
@@ -21,6 +31,96 @@ def __init__(self, enterprise_configuration, client=MoodleAPIClient):
             client=client
         )
 
+    def _transmit_create(self, channel_metadata_item_map, chunk_size=None):  # pylint: disable=arguments-differ
+        """
+        Transmit content metadata creation to integrated channel.
+        """
+        chunk_size = chunk_size if chunk_size else self.enterprise_configuration.transmission_chunk_size
+        chunk_items = chunks(channel_metadata_item_map, chunk_size)
+        transmission_limit = settings.INTEGRATED_CHANNELS_API_CHUNK_TRANSMISSION_LIMIT.get(
+            self.enterprise_configuration.channel_code()
+        )
+        for chunk in islice(chunk_items, transmission_limit):
+            serialized_chunk = self._serialize_items(list(chunk.values()))
+            try:
+                self.client.create_content_metadata(serialized_chunk)
+            except ClientError as exc:
+                if exc.status_code == 414:
+                    self._transmit_create(channel_metadata_item_map, chunk_size=LOW_CHUNK_SIZE)
```

Comment:
I realized a scenario:
1 chunk has very small descriptions/other data so it gets successfully transmitted at the current chunk size.
Then the next chunk has very wordy content_metadata and hits this error.

With the way this is constructed, I am not 100% sure if it would attempt to resubmit the successful items. I *imagine* it would not due to the _create_transmissions call, but this is an elaborate test case, so I haven't set it up.